labels/de-DE.json

```diff
@@ -594,5 +594,10 @@
   "width": "Breite",
   "wrongCountryForType": "Bitte wählen Sie ein Land für diesen Kontakt Typ unterstützt",
   "yourAccount": "Ihr Konto",
-  "yourOrderNumber": "Ihre Auftragsnummer ist <strong> {0} </ strong>."
+  "yourOrderNumber": "Ihre Auftragsnummer ist <strong> {0} </ strong>.",
+  "quoteInReview": "Bonjour, <br /> <br /> <br /> Le numéro du devis: {0} et le nom du devis: {1} ont été mis à jour. Veuillez examiner les modifications.",
+  "quoteReadyCheckout": "Bonjour, <br /> <br /> <br /> Le numéro du devis: {0} nom du devis: {1} a été mis à jour. Veuillez consulter les mises à jour en vous connectant à votre compte.",
```

Comment:
ReadyForCheckout.. change it everywhere..

Reply:
Done